### ISSUE TITLE
Password is printed to screen when prompted

### ISSUE THREAD

**Issue by Altreus** [rjbs/Dist-Zilla#459]
If you're like me and don't read docs, or like anyone else and don't want to put your username and password in plaintext files in your home directory, UploadToCPAN asks for your PAUSE credentials.

Then it echoes your password to the terminal while you type it.

It should probably not do that.

**Comment by karenetheridge** [comment]
It is not supposed to (see the 'noecho' bit):

```
    return $self->_credential('password')
        || $self->pause_cfg->{password}
        || $self->zilla->chrome->prompt_str('PAUSE password: ', { noecho => 1 });
```

**Comment by Altreus** [comment]
You're right - that's in the version I have, too, so it's not like it's missing.

I guess that Dist::Zilla::Chrome::Term doesn't correctly use `noecho` in some situations, but I'm not sure what information about my system would help determine what that situation is

**Comment by dolmen** [comment]
On which platform? I suspect this is Win32...

**Comment by Altreus** [comment]
Here's everything that seems relevant at the moment... I'm not sure what system libraries Term::ReadKey uses, if any.

    $ uname -a
    Linux dev-05 3.13.0-51-generic #84-Ubuntu SMP Wed Apr 15 12:08:34 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

    $ lsb_release -a
    No LSB modules are available.
    Distributor ID:	Ubuntu
    Description:	Ubuntu 14.04.2 LTS
    Release:	14.04
    Codename:	trusty

    $ mversion Dist::Zilla::Chrome::Term
    5.036

    $ mversion Term::ReadKey
    2.32

**Comment by karenetheridge** [comment]
I'd suggest putting together a small self-contained script using Term::ReadKey that demonstrates the issue (using code from Dist::Zilla::Chrome::Term as a guide), and filing that as a ticket against the offending module (presumably Term::ReadKey, but possibly something upstream that it's using).
